lib/types/types.go

```diff
@@ -233,3 +227,56 @@ func (d NullDuration) ValueOrZero() Duration {
 
 	return d.Duration
 }
+
+func getInt64(v interface{}) (int64, error) {
+	switch n := v.(type) {
+	case int:
+		return int64(n), nil
+	case int8:
+		return int64(n), nil
+	case int16:
+		return int64(n), nil
+	case int32:
+		return int64(n), nil
```

Comment:
I'm still baffled that multiple cases don't work for type switches 😕:

This could be much more succinct:
```go
	case int, int8, int16, int32:
	  return int64(n), nil
```

But I guess there's a good reason for it.

Reply:
You can do that, but then `n` will be of an `interface{}` type, and you can't directly cast that to `int64`... You can do it via the `reflect` package, but that'd be much slower.